app/Http/Controllers/DashboardEventController.php

```diff
@@ -62,6 +62,28 @@ protected function redirectToEventIfNoRegistrationIsPossible(Event $event)
         }
     }
 
+    /**
+     * Redirect to the event page for the given event if the unregistration is not possible
+     *
+     * @param Event $event
+     *
+     * @return \Illuminate\Http\RedirectResponse|void
+     */
+    protected function redirectToEventIfNoUnregistrationIsPossible(Event $event)
+    {
+        // check if unregistration is already open
```

Comment:
We check if the registration is already open, not if the unregistration is already open